dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsOrderBookStreamTestSuite.scala

```diff
@@ -402,12 +402,29 @@ class WsOrderBookStreamTestSuite extends WsSuiteBase {
       Using.resource((1 to 3).map(_ => mkWsOrderBookConnection(assetPair, dex1))) { wscs =>
         wscs.foreach(_.receiveAtLeastN[WsOrderBookChanges](1))
 
+        dex1.restartWithNewSuiteConfig(ConfigFactory.parseString(
+          s"""waves.dex {
+             |  blacklisted-assets  = [${assetPair.amountAsset}]
+             |}""".stripMargin
+        ).withFallback(dexInitialSuiteConfig))
         dex1.tryApi.deleteOrderBookWithKey(assetPair)
 
         val expectedMessage = WsError(
           timestamp = 0L, // ignored
-          code = OrderBookStopped.code,
-          message = s"The order book for $assetPair is stopped, please contact with the administrator"
+          code = MatcherIsStopping.code,
+          message = MatcherIsStopping.message.text
+        )
+
+        wscs.foreach { wsc =>
+          wsc.receiveAtLeastN[WsError](1).head should matchTo(expectedMessage)
+        }
+      }
+
+      Using.resource((1 to 3).map(_ => mkWsOrderBookConnection(assetPair, dex1))) { wscs =>
+        val expectedMessage = WsError(
+          timestamp = 0L, // ignored
+          code = AmountAssetBlacklisted.code,
+          message = AmountAssetBlacklisted(asset).message.text
```

Comment:
1. This test is useful and will be required in the DEX-1402
2. So let's rename it to "close a subscription when an order book is blacklisted", make it ignorable and add a comment: "// TODO DEX-1402"
3. Also we need a separate test: "can't subscribe on blocked order book". We should try to connect to a blocked order book and it is expected that we receive `AmountAssetBlacklisted`. One connection is enough.